src/shared/components/ResourceDetails/ResourceDetails.js

```diff
@@ -90,6 +92,13 @@ function ResourceDetailsRenderer(props) {
 
   const updateResourceMutation = useUpdate(props.resourceUrl);
   const deleteResourceMutation = useDelete(props.resourceUrl);
+  const prepareDisableEdit = () => {
+    if (props.disableEdit && typeof props.disableEdit === 'string') {
+      const [disableEdit] = jsonata(props.disableEdit, { resource });
+      return disableEdit ?? false;
+    }
+    return props.disableEdit;
+  };
```

Comment:
this should be in `ExtensibilityDetails`, not in general `ResourceDetails`, because ResourceDetails is used in Extensibility and also in "normal" views, where we don't use Jsonata.

Reply:
Also when it is here Busola is only reacting in Details (There is no Edit tab, it's YAML, so something is working), but when I go to the Edit I'm able to edit the resource. So if you remove changes from here and move them to the `ExtensibilityDetails` it should start working also for creation, if not, you need to adjust these changes also in the `ExtensibilityCreate` file.

Reply:
And one more thing to this solution, it's not working when I'm on the resource that doesn't contain the label, so probably JSONata is throwing `true` all the time.

![image](https://github.com/user-attachments/assets/cc0cf5c4-a931-4ca2-8c73-4cf0aa6ecdfd)

Reply:
Example which you can apply on your test cluster without the label:
```yaml
apiVersion: authentication.gardener.cloud/v1alpha1
kind: OpenIDConnect
metadata:
  name: test123
spec:
  clientID: string
  groupsClaim: groups
  issuerURL: https://onet.pl
  jwks:
    distributedClaims: true
  supportedSigningAlgs:
    - RS256
    - ES256
    - ES512
    - RS256
  usernameClaim: sub
  usernamePrefix: '-'
```